### PR TITLE
"Run A Node" Rearchitecture

### PR DESCRIPTION
**Pull request by YazzyYaz**
* Creates a separate `Devnet-2` Page with its own instructions on how validator and bridge nodes can connect to it.
* Creates an un-ordered list of options for network selection in Bridge and Validator tutorial guides.
* Splits Bridge-Validator tutorial into separate Bridge Node and Validator Node Tutorial
* Includes a config-toml guide from the networks repo for reference
* Adds code for how to always pull latest version from `celestia-app` and `celestia-node`
Screen Shot 2022-05-20 at 1 10 41 PM

### CODE REVIEW

File: docs/nodes/devnet-2.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+cp $HOME/networks/devnet-2/genesis.json $HOME/.celestia-app/config
+```
+
+Set seeds and peers:
+
+```sh
+SEEDS="74c0c793db07edd9b9ec17b076cea1a02dca511f@46.101.28.34:26656"
+PEERS="34d4bfec8998a8fac6393a14c5ae151cf6a5762f@194.163.191.41:26656"
+sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers \
+*=.*/persistent_peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:54]: Nit, but would prefer an indentation here to make it clearer.
```suggestion
    *=.*/persistent_peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
```

File: docs/nodes/devnet-2.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+Run the following command to quick-sync from a snapshot for `devnet-2`:
+
+```sh
+cd $HOME
+rm -rf ~/.celestia-app/data; \
+mkdir -p ~/.celestia-app/data;
+SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
+egrep -o ">devnet-2.*tar" | tr -d ">"); wget -O - \
+https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
+-C ~/.celestia-app/data/
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:70]:
```suggestion
rm -rf ~/.celestia-app/data
mkdir -p ~/.celestia-app/data
SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
    egrep -o ">devnet-2.*tar" | tr -d ">")
wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
    -C ~/.celestia-app/data/
```

File: docs/nodes/devnet-2.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+You can return back to where you left off in the Bridge Node guide [here](../nodes/validator-node.md#start-the-celestia-app-with-systemd)
+
+## Delegate to a Validator
+
+To delegate tokens to the the `celesvaloper` validator, as an example you can run:
+
+```sh
+celestia-appd tx staking delegate \
+celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000celes \
+--from=$VALIDATOR_WALLET --chain-id=devnet-2
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:82]:
```suggestion
celestia-appd tx staking delegate \
    celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000celes \
    --from=$VALIDATOR_WALLET --chain-id=devnet-2
```

File: docs/nodes/devnet-2.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+PEERS="34d4bfec8998a8fac6393a14c5ae151cf6a5762f@194.163.191.41:26656"
+sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers \
+*=.*/persistent_peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
+```
+
+You can return back to where you left off in the Bridge Node guide [here](../nodes/validator-node.md#configure-pruning)
+
+## Quick-Sync With Snapshot
+
+Run the following command to quick-sync from a snapshot for `devnet-2`:
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:61]: Add a disclaimer that this is almost a 100 GiB download.

File: docs/nodes/devnet-2.md (hunk 3 above)
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:82]: `VALIDATOR_WALLET` isn't set until a later step.

File: docs/nodes/devnet-2.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+```sh
+celestia-appd tx staking delegate \
+celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000celes \
+--from=$VALIDATOR_WALLET --chain-id=devnet-2
+```
+
+If successful, you should see a similar output as:
+
+```sh
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:87]:
````suggestion
```console
````

File: docs/nodes/devnet-2.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+`networks/devnet-2/celestia-node/mutual_peers.txt` to find the list of
+mutual peers
+
+For more information on `config.toml`, please navigate to [this link](../nodes/config-toml.md)
+
+```sh
+nano ~/.celestia-bridge/config.toml
+```
+
+```sh
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:121]:
````suggestion
```toml
````

File: docs/nodes/devnet-2.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+celestia-appd tx staking create-validator \
+--amount=1000000celes \
+--pubkey=$(celestia-appd tendermint show-validator) \
+--moniker=$MONIKER \
+--chain-id=devnet-2 \
+--commission-rate=0.1 \
+--commission-max-rate=0.2 \
+--commission-max-change-rate=0.01 \
+--min-self-delegation=1000000 \
+--from=$VALIDATOR_WALLET
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:157]:
```suggestion
celestia-appd tx staking create-validator \
    --amount=1000000celes \
    --pubkey=$(celestia-appd tendermint show-validator) \
    --moniker=$MONIKER \
    --chain-id=devnet-2 \
    --commission-rate=0.1 \
    --commission-max-rate=0.2 \
    --commission-max-change-rate=0.01 \
    --min-self-delegation=1000000 \
    --from=$VALIDATOR_WALLET
```

File: docs/nodes/devnet-2.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+```
+
+You will be prompted to confirm the transaction:
+
+```sh
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:162]:
````suggestion
```console
````

File: docs/nodes/devnet-2.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+confirm transaction before signing and broadcasting [y/N]: y
+```
+
+Inputting `y` should provide an output similar to:
+
+```sh
```
Comment (adlerjohn) [inline docs/nodes/devnet-2.md:168]:
````suggestion
```console
````

**Comment by YazzyYaz** [comment]
@adlerjohn resolved your suggestions and added recommendations based on your feedback

**Review by adlerjohn — APPROVED** [review]
utACK

**Pull request merged**